pymeasure/display/inputs.py

```diff
@@ -41,10 +41,20 @@ def __init__(self, parameter):
         super().__init__()
         self._parameter = None
         self.set_parameter(parameter)
+        self.set_value(parameter)
 
     def set_parameter(self, parameter):
         self._parameter = parameter
 
+    def set_value(self, parameter):
+        if hasattr(self, 'setValue'):
+            try:
+                value = parameter.value  # if parameter is not set yet, this will raise ValueError
+            except ValueError:
+                pass  # parameter is not set yet
+            else:
```

Comment:
1. The `StringInput` that inherits from `QtGui.QLineEdit` doesn't have a `setValue()` method.
2. Maybe a better name for that method would be `init_value()`.
3. I didn't want to catch an `ValueError` inside the `setValue()` call. That's why it was separated. I'll try to do it less obscure.